federation/pkg/dnsprovider/providers/coredns/rrsets.go

```diff
@@ -78,13 +77,10 @@ func (rrsets ResourceRecordSets) Get(name string) (dnsprovider.ResourceRecordSet
 		}
```

Comment:
This may not always be true. Either way, the code should be fixed to handle the rrset type correctly.